projects/core/src/checkout/store/effects/checkout.effect.ts

```diff
@@ -157,7 +180,11 @@ export class CheckoutEffects {
             new fromActions.CreatePaymentDetailsSuccess(details),
           ]),
           catchError(error =>
-            of(new fromActions.CreatePaymentDetailsFail(error))
+            of(
+              new fromActions.CreatePaymentDetailsFail(
+                makeHttpErrorSerializable(error)
```

Comment:
this is not catching the current unserializable error in payment form in checkout
- create new payment will throw the error, but this doesn't catch it

Reply:
fixed